scout/utils/acmg.py

```diff
@@ -157,38 +162,31 @@ def is_likely_benign(bs_terms, bp_terms):
     return False
 
 
-def get_acmg(acmg_terms):
-    """Use the algorithm described in ACMG paper to get a ACMG calssification
-
-    Modifying strength of a term is possible by adding a string describing its new level: "PP1_Strong" or
-    "PVS1_Moderate".
-
-    If no terms return None
-
-    Args:
-        acmg_terms(set(str)): A collection of prediction terms
-
-    Returns:
-        prediction(str): in ['uncertain_significance','benign','likely_benign',
-                             'likely_pathogenic','pathogenic']
-
+def get_acmg_criteria(acmg_terms: set) -> tuple:
```

Comment:
Refactored slightly to have one function to sort the terms, that can be reused by both the Richards and the Tavtigian classifier.